signalbot/context.py

```diff
@@ -27,6 +27,22 @@ async def send(
             mentions=mentions,
             text_mode=text_mode,
         )
+    async def edit(
```

Comment:
I don't quite understand how this works, add the timestamp of the original message and that is what gets edited? Would you consider adding an example on how to use this in the example folders?